db/migrate/20161226201452_change_tasks_timestamps_default_to_null.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTasksTimestampsDefaultToNull < ActiveRecord::Migration
+  def change
+    change_column_null :tasks, :updated_at, false
+    change_column_null :tasks, :created_at, false
+  end
+end
```

Comment:
Is there any particular reason you chose to do the adding of the timestamps and the adding of the nullability restriction as separate from the model creation?